.github/workflows/verify.yml

```
@@ -63,6 +63,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Run Docker Compose tests
+        if: ${{ ! github.event.pull_request.head.repo.fork || matrix.compose_file == 'community-docker-compose.yml' }}
```

```suggestion
        if: ! github.event.pull_request.head.repo.fork || matrix.compose_file == 'community-docker-compose.yml'
```

inside ifs expansion brackets are implicit

---

to this day it is still confusing for me where they are implicit and where not so I usually put them everywhere to be sure XD